lib/cartopy/crs.py

```diff
@@ -1640,11 +1638,23 @@ def __init__(self, central_longitude=0.0, central_latitude=0.0,
                         ('lat_0', central_latitude),
                         ('x_0', false_easting), ('y_0', false_northing)]
         super(AzimuthalEquidistant, self).__init__(proj4_params, globe=globe)
-        self._max = 2e7
+
+        # TODO: Let the globe return the semimajor axis always.
+        a = np.float(self.globe.semimajor_axis or 6378137.0)
+        b = np.float(self.globe.semiminor_axis or a)
+
+        coords = _ellipse_boundary(a * np.pi, b * np.pi,
+                                   false_easting, false_northing, 61)
+        coords = tuple(tuple(pair) for pair in coords.T)
+
+        self._boundary = sgeom.polygon.LinearRing(coords)
```

Comment:
You're probably correct; I copied this code out of the `Geostationary` projection. There are a couple of other cleanups I've thought of, so I'll put this change in that PR.